doc-content/enterprise-only/openshift/operator-subscribe-proc.adoc

```diff
@@ -8,4 +8,8 @@ To be able to deploy {PRODUCT} using operators, you must subscribe to {operator_
 . Enter your project in the OpenShift Web cluster console.
 . In the OpenShift Web console navigation panel, select  *Catalog -> OperatorHub* or *Operators -> OperatorHub*.
 . Search for *Business Automation*, select it and click *Install*.
-. On the *Create Operator Subscription* page, select your target namespace and approval strategy and then click *Subscribe* to create a subscription.
+. On the *Create Operator Subscription* page, select your target namespace and approval strategy.
++
+Set *Approval strategy* to `Automatic` to enable automatic operator updates. An operator update does not immediately update the product, but is required before you update the product. To configure automatic or manual product updates, use the settings in every particular product deployment.
```

Comment:
"To configure automatic or manual product updates, use the settings in every particular product deployment"

Can this be reworded - Such as 
Configure automatic or manual product updates using each product deployment's settings.